src/services/lnd/index.ts

```diff
@@ -462,7 +462,30 @@ const lookupPaymentByPubkeyAndHash = async ({
   }
 }
 
-const parseLndErrorDetails = (err) =>
+// A rough description of the error type we get back from the
```

Comment:
Is this comment still needed with the test? Can we just say something like:
```
// to see our assumption of how the error is constructed see the <xxx> test
```

Reply:
@nicolasburtey had originally requested it, so once he's ok with removing I don't see why not